e2e/terraform/main.tf

```
@@ -23,11 +23,13 @@ resource "google_compute_instance" "vm_instance" {
   allow_stopping_for_update = true
   metadata = {
   ssh-keys = "${var.ansible_user}:${file(var.ssh_pub_key)}"
+  startup-script-url = "startup-script-url=https://raw.githubusercontent.com/nephio-project/test-infra/main/e2e/provision/gce_init.sh"
```

I couldn't find any pointer to `startup-script-url` key into the documentation. I think the right one is [metadata_startup_script](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance#metadata_startup_script), so this will be 

```suggestion
  metadata_startup_script = file("${path.module}/../provision/gce_init.sh")
```

Maybe the provisioning is running in the background, but Terraform is not waiting for it, or it's simply not provisioning anything at all. Anyhow, [the e2e task is not performing](http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/63/e2e/1666268068000567296#1:build-log.txt%3A216).

---

Mostly I wanted to touch these files because I think I am breaking the e2e by moving files around (gce_e2e.sh -> ../e2e.sh, etc.)